aiohttp/client_reqrep.py

```diff
@@ -35,11 +35,14 @@
 
 
 class ClientRequest:
-
-    GET_METHODS = {hdrs.METH_GET, hdrs.METH_HEAD, hdrs.METH_OPTIONS}
+    GET_METHODS = {
+        hdrs.METH_GET,
+        hdrs.METH_HEAD,
+        hdrs.METH_OPTIONS,
+        hdrs.METH_TRACE,
```

Comment:
Nothing is said regarding DELETE in RFC. And also:

1. a message-body is explicitly forbidden in responses to HEAD requests (section 9, and 9.4 specifically)
2. an entity-body is explicitly forbidden in TRACE requests only, all other request types are unrestricted (section 9, and 9.8 specifically)

Should I add raise ValueError in constructor in these requests ?